integration/spark/app/src/test/java/io/openlineage/spark/agent/column/ColumnLevelLineageIcebergTest.java

```diff
@@ -158,8 +158,8 @@ void testCreateTableAsSelectWithUnion() {
             .get();
 
     assertColumnDependsOn(facet, "a", FILE, T1_EXPECTED_NAME, "a");
-    assertColumnDependsOn(facet, "a", FILE, "/tmp/column_level_lineage/db.t2", "a");
-    assertColumnDependsOn(facet, "b", FILE, "/tmp/column_level_lineage/db.t2", "b");
+    assertColumnDependsOn(facet, "a", FILE, T2_EXPECTED_NAME, "a");
```

Comment:
Would it be possible within this (or any other test) to verify that the directory exists and is non-empty? 
This would contain a real proof of data name pointing to physical location of data.

Reply:
Done. But I'm not sure if this a right place for such check, as it is not related to column lineage.

Reply:
Perhaps one of the `SparkIcebergIntegrationTest` would be better for this. Anyway it's better to have it here, than not having it at all.